feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/CameraControlsOverlay.kt

```diff
@@ -335,41 +333,42 @@ private fun CaptureButton(
 }
 
 @Composable
-private fun HdrCaptureModeToggleButton(
-    initialImageFormat: ImageOutputFormat = ImageOutputFormat.JPEG,
-    supportedImageFormats: Set<ImageOutputFormat>? = null,
-    onChangeImageFormat: (ImageOutputFormat) -> Unit = {}
+private fun CaptureModeToggleButton(
+    uiState: CaptureModeToggleUiState,
```

Comment:
With the sealed interface change, this will become:
`uiState: CaptureModeToggleUiState.Visible`